### PR TITLE
Return failure if the tidy self-tests fail.

### PR DESCRIPTION
**Pull request by Ms2ger**
This change is [Reviewable](https://reviewable.io/reviews/servo/servo/12567)

### CODE REVIEW

**Comment by Ms2ger** [comment]
Will fail until #12565 lands.

**Comment by jdm** [comment]
Whoops. R=me when appropriate.

File: python/tidy/servo_tidy_tests/test_tidy.py [hunk 1]
```diff
@@ -138,4 +138,4 @@ def test_file_list(self):
 
 def do_tests():
     suite = unittest.TestLoader().loadTestsFromTestCase(CheckTidiness)
-    unittest.TextTestRunner(verbosity=2).run(suite)
+    return 0 if unittest.TextTestRunner(verbosity=2).run(suite).wasSuccessful() else 1
```
Comment (Wafflespeanut) [inline python/tidy/servo_tidy_tests/test_tidy.py]: Wouldn't it be better if we had a name for the result?

**Pull request merged**